### ISSUE TITLE
Problems with Typoscrit config for redirects

### ISSUE THREAD

**Issue by webian** [in2code-de/femanager#270]
To redirect after the user confirm the registration, I modified the sample Typoscript this way:
```
userConfirmationRedirect = TEXT
userConfirmationRedirect {
	typolink {
		parameter = 33
		returnLast = url
	}
}
```

With this Typoscript the $target variable has already the relative url so this code doesn't make sense:
https://github.com/in2code-de/femanager/blob/3dc5830934d46c8961b0e0ed8e7eabc43915ed75/Classes/Controller/AbstractController.php#L312

That code makes sense only if $target is an uid so with a Typoscript like:
```
userConfirmationRedirect = TEXT
userConfirmationRedirect.value = 33
```

**Comment by webian** [comment]
I just found that this issue is a duplicate of #252

**Comment by spoonerWeb** [comment]
Fixed with #258.
